### ISSUE TITLE
click a city to isolate

### ISSUE THREAD

**Issue by russellsamora** [the-pudding/same-name#8]
this would be a choropleth of just that one city's score. it's tricky because it would be better to show the new scale (0-100), not probably/maybe. but i could also show the binary.

**Comment by mmcghee18** [comment]
I still like this, but could definitely live without it, it doesn't feel incomplete without it

**Comment by iblind** [comment]
yeah I think it's nice but doesn't change impression of the piece
